docker/driver-loader/docker-entrypoint.sh

```diff
@@ -85,12 +89,20 @@ while test $# -gt 0; do
 			;;
 		--http-insecure)
 			HTTP_INSECURE="true"
-			;;   
+			;;
 		--source-only)
-		    >&2 echo "Support dropped in Falco 0.37.0."
+			>&2 echo "Support dropped in Falco 0.37.0."
```

Comment:
While you are here, can you remove `--source-only` case, since it is no more supported (we just added the deprecation NOTE for Falco 0.37.0, but we can remove it now).

Reply:
Otherwise i can do that in a followup PR :)

Reply:
Done! :slightly_smiling_face: